algorithm/cgal/insetOfPolygonWithHoles.test.js

```diff
@@ -16,128 +16,136 @@ test('insetOfPolygonWithHoles/bow', (t) => {
+        ['5/28', '-61/518', '0'],
         ['1/4', '-33/370', '0'],
-        ['15/52', '-153/1591', '0'],
-        ['131/190', '-2774699/8480030', '0'],
-        ['131/190', '2774699/8480030', '0'],
+        ['9/28', '-61/518', '0'],
+        ['131/190', '-17511/53095', '0'],
+        ['131/190', '17511/53095', '0'],
       ],
       holes: [],
       plane: [0, 0, 1, 0],
     },
     {
       points: [
-        [0.3071428571428571, 0.05125815470643057, 0],
+        [0.35526315789473684, 0.08392603129445234, 0],
         [0.25, 0.039189189189189184, 0],
-        [0.19285714285714284, 0.05125815470643057, 0],
-        [-0.1394736842105263, 0.24272627788187082, 0],
-        [-0.1394736842105263, -0.24272627788187082, 0],
-        [0.19285714285714284, -0.05125815470643057, 0],
+        [0.14473684210526314, 0.08392603129445234, 0],
+        [-0.1394736842105263, 0.2476702633313673, 0],
+        [-0.1394736842105263, -0.2476702633313673, 0],
+        [0.14473684210526314, -0.08392603129445234, 0],
         [0.25, -0.039189189189189184, 0],
-        [0.3071428571428571, -0.05125815470643057, 0],
-        [0.6394736842105263, -0.24272627788187082, 0],
-        [0.6394736842105263, 0.24272627788187082, 0],
+        [0.35526315789473684, -0.08392603129445234, 0],
+        [0.6394736842105263, -0.2476702633313673, 0],
+        [0.6394736842105263, 0.2476702633313673, 0],
       ],
       exactPoints: [
-        ['43/140', '55/1073', '0'],
+        ['27/76', '59/703', '0'],
         ['1/4', '29/740', '0'],
-        ['27/140', '55/1073', '0'],
-        ['-53/380', '373739/1539755', '0'],
-        ['-53/380', '-373739/1539755', '0'],
-        ['27/140', '-55/1073', '0'],
+        ['11/76', '59/703', '0'],
+        ['-53/380', '35693/144115', '0'],
+        ['-53/380', '-35693/144115', '0'],
+        ['11/76', '-59/703', '0'],
         ['1/4', '-29/740', '0'],
-        ['43/140', '-55/1073', '0'],
-        ['243/380', '-373739/1539755', '0'],
-        ['243/380', '373739/1539755', '0'],
+        ['27/76', '-59/703', '0'],
+        ['243/380', '-35693/144115', '0'],
+        ['243/380', '35693/144115', '0'],
       ],
       holes: [],
       plane: [0, 0, 1, 0],
     },
     {
       points: [
-        [-0.0894736842105263, -0.15526915283012843, 0],
-        [0.17307692307692307, -0.004004004004004004, 0],
-        [0.19386694386694386, 0, 0],
-        [0.17307692307692307, 0.004004004004004004, 0],
-        [-0.0894736842105263, 0.15526915283012843, 0],
+        [-0.0894736842105263, -0.1631282959789057, 0],
+        [0.109375, -0.048564189189189186, 0],
+        [0.22439544807965858, 0, 0],
+        [0.109375, 0.048564189189189186, 0],
+        [-0.0894736842105263, 0.1631282959789057, 0],
       ],
       exactPoints: [
-        ['-17/190', '-826757/5324670', '0'],
-        ['9/52', '-4/999', '0'],
-        ['373/1924', '0', '0'],
-        ['9/52', '4/999', '0'],
-        ['-17/190', '826757/5324670', '0'],
+        ['-17/190', '-79189/485440', '0'],
```

Comment:
What was the change to inset? Was it the addition of segments?...or was that already there?

Reply:
The additional of parameterized segments, I think.